cpu/kinetis_common/ldscripts/kinetis-noramcode.ld

```diff
@@ -0,0 +1,257 @@
+/* RAM limits */
+__sram_start  = ORIGIN(sram);
+__sram_length = LENGTH(sram);
+__sram_end    = __sram_start + __sram_length;
+_ramcode_start = 0x0;
+_ramcode_end = 0x0;
+_ramcode_load = 0x0;
+
+/* Define the default stack size for interrupt mode. As no context is
+   saved on this stack and ISRs are supposed to be short, it can be fairly
+   small. 512 byte should be a safe assumption here */
+STACK_SIZE = DEFINED(STACK_SIZE) ? STACK_SIZE : 0x800;
```

Comment:
Just curious: 2k of stack size for the ISRs? Being very conservative here and optimize later or are these really necessary?